gax-java/gax/src/main/java/com/google/api/gax/rpc/ServerStreamingCallSettings.java

```diff
@@ -233,9 +251,9 @@ public Builder<RequestT, ResponseT> setSimpleTimeoutNoRetries(@Nonnull Duration
               .setInitialRetryDelay(Duration.ZERO)
               .setRetryDelayMultiplier(1)
               .setMaxRetryDelay(Duration.ZERO)
-              .setInitialRpcTimeout(Duration.ZERO)
+              .setInitialRpcTimeout(timeout)
               .setRpcTimeoutMultiplier(1)
-              .setMaxRpcTimeout(Duration.ZERO)
+              .setMaxRpcTimeout(timeout)
```

Comment:
I might be missing something. Does this need to change? The ServerStreamAtttemptCallable only sets a timeout if the RPC Timeout is non-zero. For this case (the simple timeout + no retry), the RPC Timeout would be zero (from the initial RPC timeout value) and no timeout would be set (which *should* cancel once we hit the total timeout).

Reply:
Ah, gotcha. I missed the part where the totalTimeout was removed. Makes sense that it uses the RpcTimeout value now.